python/examples/tools/agent.py

```diff
@@ -0,0 +1,20 @@
+import asyncio
+
+from beeai_framework.agents.bee import BeeAgent
+from beeai_framework.agents.types import BeeInput, BeeRunInput
+from beeai_framework.backend.chat import ChatModel
+from beeai_framework.memory import UnconstrainedMemory
+from beeai_framework.tools.weather.openmeteo import OpenMeteoTool
+
+
+async def main() -> None:
+    llm = ChatModel.from_name("ollama:3.1")
```

Comment:
I'm not sure this is valid, the example errors for me

Reply:
I'll double check this.

Reply:
Hmm not sure what changed if anything since writing, but it definitely errors out: `LinePrefixParserError: The generated output does not adhere to the schema.`
 
 Looks like the same issue when trying to run the `example/tools/openmeteo.py` example as well and also the `examples/agents/simple.py` example.